src/metabase/pivot/core.cljc

```diff
@@ -27,6 +27,17 @@
                 (decimal? x) (double x) ;; Handles BigDecimal
                 :else x)))
 
+(defn- ensure-is-int
+  "Convert a clojure value that may not be a type bitwise operations can use into one it can. Or throw an
+  error if it is not losslessly convertible into an integer. "
+  [x]
+  #?(:cljs x
+     :clj (cond
+            (int? x) x
+            (integer? x) (long x)
+            (decimal? x) (.longValueExact ^BigDecimal x)
+            :else x)))
```

Comment:
should we throw here? it feels a bit odd for `(ensure-is-int "123")` to return `"123"`